addons/mail/static/src/discuss/core/common/store_service_patch.js

```diff
@@ -70,21 +70,26 @@ const storeServicePatch = {
     sortMembers(m1, m2) {
         return m1.name?.localeCompare(m2.name) || m1.id - m2.id;
     },
-    /** @param {number[]} partnerIds */
+    /**
+     * @param {number[]} partnerIds
+     * @returns {Promise<import("models").Thread>}
+     */
     async startChat(partnerIds) {
         const partners_to = [...new Set([this.self.id, ...partnerIds])];
+        let newThread;
         if (partners_to.length === 1) {
-            const chat = await this.joinChat(partners_to[0], true);
-            chat.open({ focus: true, bypassCompact: true });
+            newThread = await this.joinChat(partners_to[0], true);
+            newThread.open({ focus: true, bypassCompact: true });
```

Comment:
I'm not sure if we really need to rename here, the new name is longer/more confusing.

Or maybe just `channel`? But we kind of consider groups (type) the same as chat (in naming, for example in buttons/filters/categories).

Not renaming is just less diff, too.